pegasus/sites.v3/code.org/public/global/fa/csd.haml

```diff
@@ -0,0 +1,125 @@
+---
+title: Computer Science Discoveries
+theme: farsi_theme
```

Comment:
This is causing an error. Should it be replaced with `responsive_full_width` like the other pages?